frontend/shared/browser-tests/components/header.components.ts

```diff
@@ -8,10 +8,25 @@ import {
   withinContext,
 } from '../utils/testcafe.utils';
 
-const langTranslations = {
-  fi: 'Suomeksi',
-  sv: 'På svenska',
-  en: 'In english',
+const translations = {
+  fi: {
+    login: 'Kirjaudu sisään',
+    logout: 'Kirjaudu ulos',
+    language: 'Suomeksi',
+    userInfo: (user?: User) => new RegExp(`Käyttäjä: ${user?.name ?? ''}`),
+  },
+  sv: {
+    login: 'Logga in',
+    logout: 'Logga ut',
+    language: 'På svenska',
+    userInfo: (user?: User) => new RegExp(`Användare: ${user?.name ?? ''}`),
+  },
+  en: {
+    login: 'Login',
+    logout: 'Logout',
+    language: 'In english',
```

Comment:
```suggestion
    login: 'Log in',
    logout: 'Log out',
    language: 'In English',
```

Reply:
I'm probably just nitpicking here 😅

Reply:
picked up the changes to my latest pr. Anyway translations are now made by me and will be real ones later.